cohorts/variant_filters.py

```diff
@@ -51,3 +53,45 @@ def variant_string_to_variant(variant_str, reference="grch37"):
     alt = m.group(3)
     variant = Variant(contig=chrom, start=start, ref=ref, alt=alt, ensembl=reference)
     return variant
+
+def load_ensembl_coverage(cohort, coverage_path, min_depth=30):
```

Comment:
Guess I was viewing this file as things related to depth; agree that it's a little weird since it's not a filter.